src/lib.rs

```diff
@@ -13,7 +13,9 @@
 //!
 //! ## PKCS#1 v1.5 encryption
 //! ```
-//! use rsa::{PublicKey, RsaPrivateKey, RsaPublicKey, PaddingScheme};
+//! use rsa::{RsaPrivateKey, RsaPublicKey};
+//! use rsa::pkcs1v15::{DecryptingKey, EncryptingKey};
```

Comment:
`EncryptingKey` reads quite odd, I would have expected it to be called `EncryptionKey`

Reply:
Ack